api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -145,7 +145,10 @@ message ExternalProcessor {
   // to the processing mode) if the timer expires before a matching response
   // is received. There is no timeout when the filter is running in asynchronous
   // mode. Default is 200 milliseconds.
-  google.protobuf.Duration message_timeout = 7;
+  google.protobuf.Duration message_timeout = 7 [(validate.rules).duration = {
+    lte {seconds: 3600}
```

Comment:
I think this is legit and shouldn't break anyone.

/lgtm api

Reply:
Yes, this is in ext_proc filter config, i.e, in control plane, so PGVs can help bound the configuration and not trigger Duration out-of-range exception being thrown and crash.